pkg/crd/known_types.go

```diff
@@ -77,6 +77,7 @@ var KnownPackages = map[string]PackageOverride{
 
 	"k8s.io/apimachinery/pkg/util/intstr": func(p *Parser, pkg *loader.Package) {
 		p.Schemata[TypeIdent{Name: "IntOrString", Package: pkg}] = apiext.JSONSchemaProps{
+			XIntOrString: true,
```

Comment:
little, but important detail: order must be integer, string, compare https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190425-structural-openapi.md#formal-definition

Reply:
:open_mouth: Fixed!